acceptance/bundle/artifacts/glob_exact_whl/test.toml

```diff
@@ -0,0 +1,6 @@
+Badness = 'Uploading the same wheel twice'
```

Comment:
I don't see this in the output.

Reply:
Good catch, seems like I copy-pasted test.toml. https://github.com/databricks/cli/pull/2559